server/controllers/match.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose')
```

Comment:
Instead of importing mongoose, you should import the model directly

Reply:
Corrected this. Exporting the model from models/MatchModel and importing it directly in match controller